runbotics/runbotics-orchestrator-ui/src/src-app/Actions/desktop.actions.ts

```diff
@@ -0,0 +1,192 @@
+import { IBpmnAction, Runner } from './types';
+
+// eslint-disable-next-line max-lines-per-function
+const getDesktopActions: () => Record<string, IBpmnAction> = () => ({
+    'desktopAutomation.click': {
+        id: 'desktopAutomation.click',
+        label: 'Click',
+        script: 'desktopAutomation.click',
+        runner: Runner.DESKTOP_SCRIPT,
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: 'Input',
+                        type: 'object',
+                        properties: {
+                            xCoordinate: {
+                                title: 'Target x-coordinate',
+                                type: 'number',
+                            },
+                            yCoordinate: {
+                                title: 'Target y-coordinate',
+                                type: 'number',
```

Comment:
consider that this makes passing variable, e.g. #{yCoords} impossible because variable is string